File: Scala/src/main/scala/Exibir.scala
object Exibir {

  def main(args: Array[String]) {

    val inicio = System.nanoTime()

    val text = "Olá Mundo!"

    println(text)

    val fim = System.nanoTime()

    println(fim - inicio)

    println(Runtime.getRuntime.totalMemory - Runtime.getRuntime.freeMemory)

  }

}

File: Java/src/com/senai/Exibir.java
package com.senai;

public class Exibir {

    public static void main(String[] args) {

        long inicio = System.nanoTime();

        String text = "Olá Mundo!";

        System.out.println(text);

        long fim = System.nanoTime();

        Runtime runtime = Runtime.getRuntime();

        System.out.println(fim - inicio);

        System.out.println((runtime.totalMemory() - runtime.freeMemory()));

    }

}
